src/app/app.component.ts

```diff
@@ -62,13 +63,16 @@ export class AppComponent implements OnInit {
                     this.gateway['config']['api_url'] = apiHost;
                 }
 
-                // Separate v2 users 
-                if (this.gateway['config']['ver'] === 'v2') {
+                // Separate v1, v2, v3 users 
+                if (this.gateway['config']['ver'] === 'v3') {
+                    this.version = 'v2';
+                    this.uuid = this.gateway['config']['uuid'];
+                } else if (this.gateway['config']['ver'] === 'v2') {
```

Comment:
@arajkumar this is how we are managing fallback to v2. With this, we set UUID and then use this UUID to show the Snyk Token button